app/presenters/character_presenter.rb

```diff
@@ -8,7 +8,8 @@ def initialize(character)
   def as_json(options={})
     return {} unless character
 
-    char_json = character.as_json_without_presenter(only: [:id, :name, :screenname])
+    char_json = character.as_json_without_presenter(only: [:id, :name, :screenname, :selector_name])
+    char_json[:selector_name] = character.selector_name
```

Comment:
I'm confused – this looks duplicated or something? (what's the functionality of both the lines?)